lib/flipper/railtie.rb

```diff
@@ -2,11 +2,11 @@ module Flipper
   class Railtie < Rails::Railtie
     config.before_configuration do
       config.flipper = ActiveSupport::OrderedOptions.new.update(
-        env_key: "flipper",
-        memoize: true,
-        preload: true,
-        instrumenter: ActiveSupport::Notifications,
-        log: true
+        env_key: ENV.fetch('FLIPPER_ENV_KEY', 'flipper'),
+        memoize: ENV.fetch('FLIPPER_MEMOIZE', 'true') == 'true',      
+        preload: ENV.fetch('FLIPPER_PRELOAD', 'true') == 'true',
```

Comment:
I'd like to see the casting here be a little more robust. Maybe use `ActiveModel::Type::Boolean.new.cast(value)`?

Reply:
Well... this is a railtie. Probably safe to assume ActiveModel is loaded? That being said string comparison is pretty robust, no? No need to over complicate things.

Reply:
For `preload`, we can provide an array... do we want to support that via the ENV?

Reply:
Bumped to `casecmp` to catch `False`